spec/passaporte_web/notification_spec.rb

```diff
@@ -204,19 +204,19 @@
   describe "#destroy" do
     let(:notification) { described_class.new(body: 'novinha', destination: 'a5868d14-6529-477a-9c6b-a09dd42a7cd2', scheduled_to: '2013-04-06') } # 2.days.from_now
     it "should return false if the record is not persisted" do
-      notification.destroy.should be_false
+      expect(notification.destroy).to be_falsy
       notification.errors.should == {message: 'notification not persisted yet'}
     end
     it "should destroy the notification on PassaporteWeb if the notification has not been read and is scheduled", vcr: true do
-      notification.save('application').should be_true
+      expect(notification.save('application')).to be_truthy
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.